### DISCUSSION TITLE
Setup/Running Redocusaurus on Vercel

### DISCUSSION THREAD

**Discussion by alixodendhal** [rohit-gohri/redocusaurus#373] [Q&A] [answered]
I know this isn't for generic support...but I am at the end of my rope...

I am trying to setup/create redocusaurus using vercel (like: https://redocusaurus.vercel.app/) 

But it isn't working. No redoc components are rendering. 
I might be dumb or missing something obvious. 

**What am I doing wrong?**

### Steps I took

1. Setup docusaurus on vercel (generic import)
2. Added redoc to the dependencies [package.json](https://github.com/alixodendhal/docusaurus/blob/redocusaurus/package.json)
3. Added the redocusaurus config to the presets in [docusaurus.config.js](https://github.com/alixodendhal/docusaurus/blob/redocusaurus/docusaurus.config.js#L56)
4. Run a Build.... 

When I check my app: https://redocusaurus-alix.vercel.app/ 
I don't get any redoc components. `/api` does not load anything. 

All I want is to get this up and running like https://redocusaurus.vercel.app/
I know it's possible but I am not sure where I am going wrong.

If anyone can point me in the right direction I would appreciate it.

**Comment by rohit-gohri** [comment]
Hey, are you sure the domain is pointing to the branch where you are making changes? I don't see the default branch having this changes

**Reply by alixodendhal** [reply]
I promoted the preview branch to prod, which should be the same on the build/vercel side.

![image](https://github.com/user-attachments/assets/a4667afd-4c0b-4955-be9f-31f86034b329)
---
I merged it anyway just clear up any confusion. 
It is still the same. 
![image](https://github.com/user-attachments/assets/3e1bd42f-5739-486f-8647-183a5bfb9c50)

https://redocusaurus-alix.vercel.app/

**Reply by rohit-gohri** [reply, marked as answer]
Hi, you are using docusaurus v2  which is quite old - https://github.com/alixodendhal/docusaurus/blob/4d3b78c4ffbda64d1153490b59aa38e295a55f58/package.json#L15-L16 

The current version of redocusaurus only supports v3+, you can see it in the peer dependencies - https://github.com/rohit-gohri/redocusaurus/blob/5d0aaf3d49245bbac64e18bd8f81222f5ab46951/packages/redocusaurus/package.json#L47-L50

**Reply by alixodendhal** [reply]
Ah, thanks. 
I didn't realize the vercel template was such an old one. 
I rebuilt from scratch and got it working.
